cmd/cloudFoundryDeleteService.go

```diff
@@ -0,0 +1,71 @@
+package cmd
+
+import (
+	"github.com/SAP/jenkins-library/pkg/command"
+	"github.com/SAP/jenkins-library/pkg/log"
+)
+
+func cloudFoundryDeleteService(CloudFoundryDeleteServiceOptions cloudFoundryDeleteServiceOptions) error {
+
+	c := command.Command{}
+
+	// reroute command output to logging framework
+	c.Stdout(log.Entry().Writer())
+	c.Stderr(log.Entry().Writer())
+
+	cloudFoundryLogin(CloudFoundryDeleteServiceOptions, &c)
+	err := cloudFoundryDeleteServiceFunction(CloudFoundryDeleteServiceOptions.CfServiceInstance, &c)
+	if err != nil {
+		log.Entry().
+			WithError(err).
+			Fatal("Failed to delete Service")
+		cloudFoundryLogout(&c)
```

Comment:
`Fatal` returns with `os.Exit(1)` thus `cloudFoundryLogout(&c)` is not called.
Maqybe just move one line up that it is called before the `Fatal` call

Reply:
Oh sorry I got that wrong. Changed it accordingly now.